src/uri/uresource.rs

```diff
@@ -11,9 +11,8 @@
  * SPDX-License-Identifier: Apache-2.0
  ********************************************************************************/
 
-use crate::uprotocol::uri::UResource;
-
-pub use crate::uri::validator::ValidationError;
+use crate::errors::ValidationError;
```

Comment:
needs to be `pub use ...` FMPOV
see previous comment ...

Reply:
Done